sdk/ai/azure-ai-projects/tests/agents/test_agents_client.py

```diff
@@ -15,8 +15,24 @@
 import time
```

Comment:
These are very long test files. Please consider splitting up into several files, based on API groups being tested. Does not have to be done in this PR.

Reply:
Sounds good